localization/languages/ca.json

```diff
@@ -0,0 +1,261 @@
+    },
+    "buttonReaderSettings": "Configuració del lector",
+    "buttonReaderLightTheme": "Clar",
+    "buttonReaderSepiaTheme": "Sepia",
+    "buttonReaderDarkTheme": "Fosc",
+    "openReaderView": "Obrir sempre en vista de lector",
+    "autoRedirectBannerReader": "Obrir sempre articles d'aquest lloc en vista de lector?",
+    "buttonReaderRedirectYes": "Sí",
+    "buttonReaderRedirectNo": "No",
+    "articleReaderView": "Article original",
+    "downloadCancel": "Cancelar",
+    "downloadStateCompleted": "Completat",
+    "downloadStateFailed": "Fallat",
+    "updateNotificationTitle": "Hi ha una nova versió de Min disponible",
+    "settingsPasswordAutoFillHeadline": "Autocompletat de contrasenyes",
+    "settingsSelectPasswordManager": "Tria un dels gestors de contrasenyes suportats:",
+    "keychainViewPasswords": "Veure contrasenyes desades",
+    "passwordManagerSetupHeading": "Acaba de configurar %p per utilitzar l'autocompletat",
+    "passwordManagerSetupStep1": {
+      "unsafeHTML": "Primer, <a id='password-manager-setup-link'></a> i extreu-ho pel teu sistema."
+    },
+    "passwordManagerInstallerSetup": {
+      "unsafeHTML": "Descarrega <a id='password-manager-setup-link-installer'></a> i arrossega el fitxer a la caixa de sota:"
+    },
+    "passwordManagerSetupLink": "descarrega l'eina CLI de %p",
+    "passwordManagerSetupLinkInstaller": "l'instal·lador CLI de %p",
+    "passwordManagerSetupStep2": "Després, arrossega l'eina a la caixa de sota:",
+    "passwordManagerSetupDragBox": "Arrossega l'eina aquí",
+    "passwordManagerSetupInstalling": "Instal·lant...",
+    "passwordManagerBitwardenSignIn": "Per connectar el teu compte Bitwarden, ves a vault.bitwarden.com/#/settings/account, desplaça't fins al final, i tria \"Veure clau API\". Després, enganxa els valors als camps de sota.",
+    "passwordManagerSetupSignIn": "Inicia sessió al teu gestor de contrasenyes per començar a utilitzar l'autocompletat. Les teves credencials no es desaran enlloc dins de Min.",
+    "disableAutofill": "Deshabilitar autocompletat",
+    "passwordManagerSetupUnlockError": "No s'ha pogut desbloquejar la botiga de contrasenyes: ",
+    "passwordManagerSetupRetry": "Assegura't d'utilitzar el fitxer correcte i d'introduir una contrasenya correcta. Pots tornar a intentar-ho arrossegant l'eina aquí de nou.",
+    "passwordManagerUnlock": "Introdueix la teva contrasenya per utilitzar l'autocompletat:",
+    "passwordManagerUnlocking": "Desbloquejant...",
+    "passwordManagerUnlockError": "Hi ha hagut un error desbloquejant la botiga de contrasenyes.",
+    "passwordManagerEnable": "Habilita el suport de Bitwarden a les preferències per començar a utilitzar l'autocompletat.",
+    "passwordManagerGetStarted": "Per començar a utilitzar l'autocompletat, ",
+    "passwordManagerGoToSettings": "ves a les preferències",
+    "passwordManagerSetup": "Configura un gestor de contrasenyes",
+    "passwordManagerUnlockErrorRetry": "Torna a intentar-ho"
```

Comment:
These don't match the strings listed at the end of the English source: https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json - where did they come from?

Reply:
TBH I don't know. I've checked again en-US and also found few missing keys.

Reply:
Thanks for updating. Just to confirm, did you write all these translations yourself, or are they machine translated (eg chatGPT)?

Reply:
I used automatic translation and then skimmed the result for inconsistencies (verb tense, formal/informal use, etc.). At first glance, it looked good to me, but it's pretty obvious I missed some things. Let me doublecheck one by one to be sure everything fits the quality standards it deserves.

Reply:
OK, sounds good, once you're done I'll re-review.

Reply:
I've double checked and applied few fixes to use a more natural language.